### PR TITLE
Fix most of pep8 issues in pyanaconda/modules subfolder

### PR DESCRIPTION
**Pull request by rvykydal**
The PR does not handle controversal:
- W503 line break before binary operator
- W504 line break after binary operator
(which are ignored by default by pycodestyle checker)
and
- E402 module level import not at top of file
which is perhaps a candidate for ignoring in config file?

After handling these (or just E402) it should be possible to add
pep8 check on pyanaconda/modules to our ci tests - as a start.

### CODE REVIEW

**Comment by rvykydal** [comment]
> After handling these (or just E402) it should be possible to add
> pep8 check on pyanaconda/modules to our ci tests - as a start.

You can use pep8 check added in https://github.com/rhinstaller/anaconda/pull/2283 to look yourself of what is missing.

File: pyanaconda/modules/storage/kickstart.py [hunk 1]
```diff
@@ -152,26 +152,32 @@ def parse(self, args):
...
         if tg.iface:
             if not wait_for_network_devices([tg.iface]):
-                raise KickstartParseError(lineno=self.lineno,
-                        msg=_("Network interface \"%(nic)s\" required by iSCSI \"%(iscsiTarget)s\" target is not up.") %
-                             {"nic": tg.iface, "iscsiTarget": tg.target})
+                raise KickstartParseError(
+                    lineno=self.lineno,
+                    msg=_("Network interface \"%(nic)s\" required by iSCSI \"%(iscsiTarget)s\" "
+                          "target is not up.") % {"nic": tg.iface, "iscsiTarget": tg.target}
+                )
```
Comment (VladimirSlavik) [inline pyanaconda/modules/storage/kickstart.py]: The splitting of strings and chaining with formatting arguments looks rather confusing.

**Comment by VladimirSlavik** [comment]
Thank you very much for this, I would really like to see PEP8 mandatory and automated.

File: pyanaconda/modules/network/network_interface.py [hunk 2]
```diff
@@ -272,7 +272,8 @@ def NetworkDeviceConfigurationChanged(self):
         """
         return self.implementation.network_device_configuration_changed()
 
-    def GetDracutArguments(self, iface: Str, target_ip: Str, hostname: Str, ibft: Bool) -> List[Str]:
+    def GetDracutArguments(self, iface: Str, target_ip: Str,
+                           hostname: Str, ibft: Bool) -> List[Str]:
```
[the remark this answers is not in the document]
Comment (rvykydal) [reply pyanaconda/modules/network/network_interface.py]: Done

File: pyanaconda/modules/network/nm_client.py [hunk 3]
```diff
@@ -1181,7 +1182,8 @@ def _get_dracut_znet_argument_from_connection(connection):
             argument = "rd.znet={},{}".format(nettype, subchannels)
             options = wired_setting.get_s390_options()
             if options:
-                argument += ",{}".format(','.join("{}={}".format(key, value) for key, value in options.items()))
+                argument += ",{}".format(','.join("{}={}".format(key, value)
+                                                  for key, value in options.items()))
```
[the remark this answers is not in the document]
Comment (rvykydal) [reply pyanaconda/modules/network/nm_client.py]: Changed, hope it is less confusing.

File: pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py [hunk 4]
```diff
@@ -427,7 +427,7 @@ def _execute_partition_data(self, storage, data, partition_data):
 
             # Get the version of LUKS and PBKDF arguments.
             partition_data.luks_version = partition_data.luks_version \
-                                          or storage.default_luks_version
+                or storage.default_luks_version
```
[the remark this answers is not in the document]
Comment (rvykydal) [reply pyanaconda/modules/storage/partitioning/custom/custom_partitioning.py]: It does, changed.

File: pyanaconda/modules/storage/iscsi/iscsi_interface.py [hunk 5]
```diff
@@ -96,7 +96,8 @@ def ReloadModule(self):
         """
         self.implementation.reload_module()
 
-    def DiscoverWithTask(self, portal: Structure, credentials: Structure, interfaces_mode: Str) -> ObjPath:
+    def DiscoverWithTask(self, portal: Structure, credentials: Structure,
+                         interfaces_mode: Str) -> ObjPath:
```
[the remark this answers is not in the document]
Comment (rvykydal) [reply pyanaconda/modules/storage/iscsi/iscsi_interface.py]: They would, fixed.

File: pyanaconda/modules/storage/kickstart.py (hunk 1 above)
Comment (rvykydal) [reply pyanaconda/modules/storage/kickstart.py]: How about using format() (keeping hints for translators)- does it look better?

File: pyanaconda/modules/storage/iscsi/iscsi_interface.py (hunk 5 above)
[the remark this answers is not in the document]
Comment (rvykydal) [reply pyanaconda/modules/storage/iscsi/iscsi_interface.py]: Not yet

[the remark this answers is not in the document]
Comment (rvykydal) [reply pyanaconda/modules/storage/iscsi/iscsi_interface.py]: Now it should be fixed. I modified also one other similar method from the class for consistency (LoginWithTask).

File: pyanaconda/modules/storage/kickstart.py (hunk 1 above)
Comment (VladimirSlavik) [reply pyanaconda/modules/storage/kickstart.py]: That looks great! Thank you!

File: pyanaconda/modules/network/nm_client.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (VladimirSlavik) [reply pyanaconda/modules/network/nm_client.py]: Works for me now :) Thank you!

**Pull request merged**